kata-deploy/kata-cleanup.yaml

```diff
@@ -18,7 +18,7 @@ spec:
           kata-containers.io/kata-runtime: cleanup
       containers:
       - name: kube-kata-cleanup
-        image: katadocker/kata-deploy
+        image: katadocker/kata-deploy:1.1.0
```

Comment:
1.1.0 or 1.2.0 ?

Reply:
same for the rest of  references.

Reply:
1.1.0.  The format changed in 1.2.0, so we need existing dockerfile for the yaml that are in repo today.  Once we have this PR landed (ie, a 1.2.0 dockerfile available), we'll update the yamls to make use of this (at which point latest should be fine again).